client/state/sites/reducer.js

```diff
@@ -27,15 +29,18 @@ export function items( state = {}, action ) {
 				[ action.site.ID ]: action.site
 			} );
 		case SERIALIZE:
-			// scrub _events, _maxListeners, and other misc functions
+			//TODO: site objects are being decorated in SitesList in lib/sites/sites-list/index.js
+			//with either lib/site/index.js or lib/site/jetpack.js
```

Comment:
> Otherwise, maybe we should consider performing the site object cleanup from the SITE_RECEIVE reducer handler

This sounds good as a first step, and move where SITE_RECEIVE is being fired for #2757 Some of the sites-list lifecycle changes might be tricky and would be better in a separate PR.